packages/block-library/src/search/index.php

```diff
@@ -179,12 +179,20 @@ function render_block_core_search( $attributes ) {
 	if ( $is_expandable_searchfield ) {
 		$aria_label_expanded  = __( 'Submit Search' );
 		$aria_label_collapsed = __( 'Expand search field' );
+		$form_context         = data_wp_context(
```

Comment:
What is this function? Are we sure the naming is correct? It's not clear to me what it does based on its name.

Reply:
From what I understood, it was included as part of the Interactivity API server directive processing: [link](https://github.com/WordPress/wordpress-develop/blob/8ee7651d0e953f0bdc5705bb5359f4067fefa89b/src/wp-includes/interactivity-api/interactivity-api.php#L142-L166).

@DAreRodz @c4rl0sbr4v0 Do you know if the name was discussed at some point? Should it be prefixed by `wp_interactivity_` as the other functions?

Reply:
There was a discussion, but rather for using a function to generate a context that naming it:
https://github.com/WordPress/gutenberg/discussions/53102#discussioncomment-7208346

If it is not clear, we can refactor it to `wp_interactivitity_create_data_wp_context`. Or `wp_interactivity_create_context_directive` or `wp_interactivity_print_context_directive`.

What do you think?

Let's do some pings to join the discussion, sorry for that 😅 
@ryanwelcher , @swissspidy , @westonruter , @youknowriad , @gziolo

Reply:
An inline thread on a closed Gutenberg pull request is really not the best place to discuss naming of WP core functions that are already in core... Can we move this elsewhere with more visibility, like a Trac ticket?

I agree that the name is not the best. Unfortunately when reviewing https://github.com/WordPress/wordpress-develop/pull/5953 this wasn't more obvious to me, so thanks for raising this. It's definitely an outlier as the only unprefixed function in this file.

Something like `wp_interactivity_get_context_attribute`, `wp_interactivity_context_attr` would be more indicative of what the function does for sure.

Reply:
Ticket created: https://core.trac.wordpress.org/ticket/60575#ticket